src/test/java/de/uhd/ifi/se/decision/management/jira/persistence/TestConfigPersistenceManager.java

```diff
@@ -292,4 +290,215 @@ public void testSetSecretForOAuth() {
 	public void testGetSecretForOAuth() {
 		assertNotNull(ConfigPersistenceManager.getSecretForOAuth());
 	}
+
+	@Test
+    public void testSetWebhookUrlNullNull(){
+	    ConfigPersistenceManager.setWebhookUrl(null, null);
+    }
+
+    @Test
+    public void testSetWebhookUrlFilledNull(){
+        ConfigPersistenceManager.setWebhookUrl("TEST", null);
+    }
+
+    @Test
+    public void testSetWebhookUrlNullFilled(){
+        ConfigPersistenceManager.setWebhookUrl(null, "http://true");
+    }
+
+    @Test
+    public void testSetWebhookUrlFilledFilled(){
+        ConfigPersistenceManager.setWebhookUrl("TEST", "http://true");
```

Comment:
Why do these Testcases don't need an assert statement?

Reply:
Yes, there should be assert statements. @dekome , is it possible to rework the mock classes so that setting has an effect?